styles.css

```diff
@@ -203,6 +214,25 @@ video {
   background: linear-gradient(135deg, red, orange , yellow, green, cyan, blue, violet);
 }
 
+/* FAQ Section */
+
+#FAQ {
```

Comment:
I want to use classes here instead of ids, but not important enough to block the PR. will fix later :)